packages/botfuel-dialog/src/brains/brain.js

```diff
@@ -26,8 +26,11 @@ class Brain {
    * @constructor
    * @param {Object} config - the config
    */
-  constructor(config = { conversationDuration: 86400000 }) {
-    this.conversationDuration = config.conversationDuration;
+  constructor(config) {
```

Comment:
this should be handled by the default config shouldn't it?

Reply:
I agree but this breaks https://github.com/Botfuel/botfuel-dialog/blob/master/packages/botfuel-dialog/tests/brains/brains.test.js#L32

I change the tests so that brain constructors are given the default config?

Reply:
done